src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
```

Comment:
You can simplify this by 
- setting `margin-right: 4px` to all stars
- setting `margin-right: 0` to `:last-child` star

It is not critical, but in most cases simpler is better.